packages/web/src/components/gcds-file-uploader/gcds-file-uploader.tsx

```diff
@@ -199,24 +185,22 @@ export class GcdsFileUploader {
   @Event() gcdsFileUploaderChange: EventEmitter;
 
   handleChange = e => {
-    if (this.changeHandler) {
-      this.changeHandler(e);
-    } else {
-      const filesContainer: string[] = [];
-      const files = e.target.files;
+    const filesContainer: string[] = [];
+    const files = e.target.files;
 
-      for (let i = 0; i < files.length; i++) {
-        filesContainer.push(files[i].name);
-      }
+    for (let i = 0; i < files.length; i++) {
+      filesContainer.push(files[i].name);
+    }
```

Comment:
Could you use `map` instead of `for` to iterate over the files to simplify the code?